kube/helper.py

```diff
@@ -142,6 +143,11 @@ def get_secret(self, name: str, namespace: str) -> V1Secret:
                 return None
         return secret
 
+    def list_secrets(self, namespace: str):
+        secrets: V1SecretList = self.core_api.list_namespaced_secret(namespace=namespace)
+        names = [s.metadata.name for s in secrets.items if s.metadata.name.startswith('cc-')]
```

Comment:
maybe factor-out `cc-`-prefix (I assume there is a second place in our codebase where this is also defined?)

Reply:
done.